src/toolkits/drawing_classifier/drawing_classifier.hpp

```diff
@@ -209,6 +214,8 @@ class EXPORT drawing_classifier : public ml_model_base {
 
   END_CLASS_MEMBER_REGISTRATION
 
+  static const size_t DRAWING_CLASSIFIER_VERSION;
```

Comment:
Let's put public constants at the top of the `public` section

https://google.github.io/styleguide/cppguide.html#Declaration_Order

Reply:
Sure!

Reply:
done.